plugin/storage/cassandra/scylladbv001-schema.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
how is this different from cassandra schema script? Can we instead parameterize that one?